### PR TITLE
Add second command port that doesn't close with `SuppressCommandPort`.

### PR DESCRIPTION
**Pull request by tylerkaraszewski**
### Details
To support escalations on the command port, and all future inter-cluster commands, we need a port to which we can send commands that does not close when traffic gets high. Historically, we close the command port on leader to keep it from servicing read traffic when it's overloaded. This works because commands are escalated to leader on the node port. In a world where traffic escalates to leader on the command port, we either need to leave this open all the time (risky), or have a second port that stays open to cluster traffic even if the public port is closed. This change implements this additional port, which acts identically to the main command port, except that it doesn't close with `SuppressCommandPort`.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/197487 https://github.com/Expensify/Expensify/issues/193926

### Tests
Tests updated, auth and bedrock tests run against the latest changes.

### CODE REVIEW

File: sqlitecluster/SQLiteNode.h
Comment (bondydaa) [inline sqlitecluster/SQLiteNode.h:45]: NAB but any reason for the line breaks?

File: sqlitecluster/SQLiteNode.h
Comment (bondydaa) [inline sqlitecluster/SQLiteNode.h:111]: could you also leave a _why_ here, looking at where it's used I think this is to ensure the port is set to the `commandPort` passed here https://github.com/Expensify/Bedrock/blob/2e115c49fea887f70ed831cb1e5c201d576a86d9/sqlitecluster/SQLiteNode.cpp#L115 which is the new `_commandPortPrivate`?

File: BedrockServer.cpp
Comment (tylerkaraszewski) [reply BedrockServer.cpp:1252]: Breaking how?

File: sqlitecluster/SQLiteNode.h
Comment (tylerkaraszewski) [reply sqlitecluster/SQLiteNode.h:45]: Legibility is all.

File: BedrockServer.cpp
Comment (bondydaa) [reply BedrockServer.cpp:1252]: like if we didn't also change auth at the time as we deploy these changes auth wouldn't start back up right? and so also anyone who updates bedrock needs to change whatever plugins they have to pass these 2 new arguments otherwise they won't work?

**Review by bondydaa — APPROVED** [review]
as far as I understand the need for the new ports I think this looks good but I'd wait for another approval too before merging just to be sure 😅

**Review by rafecolton — CHANGES_REQUESTED** [review]

**Review by rafecolton — APPROVED** [review]

**Pull request merged**
